jena-core/src/main/java/org/apache/jena/rdf/model/impl/RDFWriterFImpl.java

```diff
@@ -69,63 +72,15 @@ public RDFWriter getWriter(String lang) {
         }
     }
 
-    /**
-     * Use RIOT to add custom RDF parsers. See
-     * {@code RDFWriterRegistry.registerLang}
-     * 
-     * @deprecated Register with RIOT.
-     */
-    @Override
-    @Deprecated
-    public String setWriterClassName(String lang, String className) {
-        return setBaseWriterClassName(lang, className);
-    }
-
-    /**
-     * Use RIOT to add custom RDF parsers. See
-     * {@code RDFWriterRegistry.registerLang}
-     * 
-     * @deprecated Register with RIOT.
-     */
-    @Deprecated
-    public static String setBaseWriterClassName(String lang, String className) {
-        if ( rewiredAlternative != null )
-            Log.error(RDFWriterFImpl.class, "Rewired RDFWriterFImpl2 - configuration changes have no effect on writing");
-        String oldClassName = currentEntry(lang);
-        try {
-            @SuppressWarnings("unchecked")
-            Class<? extends RDFWriter> newClass = (Class<? extends RDFWriter>)Class.forName(className, false,
-                                                                                            Thread.currentThread().getContextClassLoader());
-            custom.put(lang, newClass);
-            return oldClassName;
-        }
-        catch (ClassNotFoundException e) {
-            throw new ConfigException("Reader not found on classpath", e);
-        }
-        catch (Exception e) {
-            throw new JenaException(e);
-        }
-    }
-
-    @Override
-    public void resetRDFWriterF() {
-        reset();
-    }
-
-    @Override
-    public String removeWriter(String lang) throws IllegalArgumentException {
-        return remove(lang);
-    }
-
-    static { // static initializer - set default readers
+    static { 
         reset();
     }
 
     private static void reset() {
         Class<? extends RDFWriter> rdfxmlWriter = org.apache.jena.rdfxml.xmloutput.impl.Basic.class;
         Class<? extends RDFWriter> rdfxmlAbbrevWriter = org.apache.jena.rdfxml.xmloutput.impl.Abbreviated.class;
         Class<? extends RDFWriter> ntWriter = org.apache.jena.rdf.model.impl.NTripleWriter.class;
-        Class<? extends RDFWriter> ttlWriter = org.apache.jena.n3.N3TurtleJenaWriter.class;
+//        Class<? extends RDFWriter> ttlWriter = org.apache.jena.n3.N3TurtleJenaWriter.class;
```

Comment:
Can be removed too? And commented code below as well?

Reply:
OK - my original attempt was merely comment out the wiring in of the writer leaving the code ... but turned into removal (we have git!) when I found all the "deprecated" that didn't actually do anything anymore.

Thanks.